### ISSUE TITLE
What should be provided in an alert vs aria-valuetext (or other attribute)

### ISSUE THREAD

**Issue by jessegreenberg** [phetsims/a11y-research#127]
This came from https://github.com/phetsims/john-travoltage/issues/318. That issue was about using aria-valuetext instead of aria-live as a workaround for poor performance with JAWS and FF, where aria-live alerts were dropped by the AT. But in that issue @emily-phet said

> As we go forward, we should explore - and set some guidelines - for what content is provided as alert, and what is provided as aria-valuetext.

This issue is to track that portion.

**Comment by terracoda** [comment]
The announcement by Microsoft that Edge will be a Chromium browser might be relevant to this issue. Posting the announcement here just in case: [Microsoft Edge: Making the web better through more open source collaboration](https://blogs.windows.com/windowsexperience/2018/12/06/microsoft-edge-making-the-web-better-through-more-open-source-collaboration/)
